commerce/domain/promotion/cart/calculator/classes/PromotionCalculatorSample.cls

```diff
@@ -0,0 +1,98 @@
+ /**
+  * @description This sample is for the situations where Promotion Calculation needs to be extended or overridden via the
+  * extension point for the Promotion Calculator. You are expected to refer this and write your own implementation.
+  * This class must extend the CartExtension.PromotionsCartCalculator class to be processed.
+  */
+public with sharing class PromotionCalculatorSample extends CartExtension.PromotionsCartCalculator {
```

Comment:
Looks good, other than the fact that tests are missing
Please add them in the subsequent PR. The apex classes need a code coverage of 80%